third_party/googletest/src/googletest/include/gtest/internal/gtest-internal.h

```diff
@@ -608,11 +608,11 @@ class GTEST_API_ TypedTestSuitePState {
   bool AddTestName(const char* file, int line, const char* case_name,
                    const char* test_name) {
     if (registered_) {
-      fprintf(stderr,
+      posix::PrintF(
```

Comment:
I left a comment on a previous review : Would it be possible to do this with a 

```
#define fprintf(a, ...) posix::Printf(__VA_ARGS) 
```

at the top of this file ? Remembering to #undef at the end.

Reply:
Clean up bug - b/288165638

(this was copied over as is from previous rebase)